vhdbuilder/packer/test/vhd-content-test.ps1

```diff
@@ -0,0 +1,189 @@
+<#
+    .SYNOPSIS
+        verify the content of Windows image built
+    .DESCRIPTION
+        This script is used to verify the content of Windows image built
+#>
+
+param (
+    $containerRuntime
+)
+
+# TODO(qinhao): we can share the variables from configure-windows-vhd.ps1
+$global:containerdPackageUrl = "https://marosset.blob.core.windows.net/pub/containerd/containerd-0.0.87-public.zip"
+
+function Compare-AllowedSecurityProtocols
+{
+
+    $allowedProtocols = @()
+    $insecureProtocols = @([System.Net.SecurityProtocolType]::SystemDefault, [System.Net.SecurityProtocolType]::Ssl3)
+
+    foreach ($protocol in [System.Enum]::GetValues([System.Net.SecurityProtocolType]))
+    {
+        if ($insecureProtocols -notcontains $protocol)
+        {
+            $allowedProtocols += $protocol
+        }
+    }
+    if([System.Net.ServicePointManager]::SecurityProtocol -ne $allowedProtocols) {
+        Write-Error "allowedSecurityProtocols '$([System.Net.ServicePointManager]::SecurityProtocol)', expecting '$allowedProtocols'"
+        exit 1
+    }
+    
+}
+
+function Test-FilesToCacheOnVHD
```

Comment:
If we can reuse the file list in configure-windows-vhd, it should be better.

Reply:
Yes, that was my idea, so that we don't have to change two parts whenever a resource is changed, the limitation so far is `config-windows-vhd.ps1` is executable and used by packer, we can:
1. source(execute) `config-windows-vhd.ps1`  in `vhd-content-test.ps1`, but `az vm run-command` is not able locate other files except the script to be executed.
2. `sed` the shared variables into  `config-windows-vhd.ps1`, that's too hacky

I'd leave a TODO here.